TestsCalculator/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace TestsCalculator.Pages
+{
+    public class SettingsPage
+    {
+        private IWebDriver _driver;
+
+        public SettingsPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public IWebElement DateFormat => _driver.FindElement(By.XPath("//th[text()=\"Date format:\"]"));
+        public SelectElement DateFormatOptions => new SelectElement(_driver.FindElement(By.Id("dateFormat")));
```

Comment:
I would name it shorter:
```suggestion
        public SelectElement DateFormat => new SelectElement(_driver.FindElement(By.Id("dateFormat")));
```